src/master/develop/sql/transactions.md

```diff
@@ -157,6 +153,42 @@ def main():
                   # my fellow time traveller
 
 
+    # interactive session
+    with client.openManagedSession(LOGIN, PASSWORD, database = DB) as session:
+        transaction = session.newTx()
+        for _ in range(3):
+            uidNow1 = str(uuid4())
+            uidNow2 = str(uuid4())
+            transaction.sqlExec("""INSERT INTO example (uniqueID, value, created) 
+                VALUES (@uid1, 'test130', NOW()), (@uid2, 'test131', NOW());""", {"uid1": uidNow1, "uid2": uidNow2})
+        resp = transaction.commit()
+
+        transaction = session.newTx()
+        resultsSize = getResultsList(transaction)
+        print("RESULTS SIZE", resultsSize) # Results size is +6 becasue we commited new transaction
+
+        for _ in range(3):
+            uidNow1 = str(uuid4())
+            uidNow2 = str(uuid4())
+            transaction.sqlExec("""INSERT INTO example (uniqueID, value, created) 
+                VALUES (@uid1, 'test130', NOW()), (@uid2, 'test131', NOW());""", {"uid1": uidNow1, "uid2": uidNow2})
+
+        resultsSize = getResultsList(transaction)
+        print("RESULTS SIZE", resultsSize)  # Results size is +6 becasue we still not commited transaction
+
+        transaction.rollback()
+
+        transaction = session.newTx()
+
+        resultsSize = getResultsList(transaction)  
+        print("RESULTS SIZE", resultsSize) # Results size is -6 becasue we rollback transaction
+        
+def getResultsList(transaction):
```

Comment:
Name of this function is not intuitive to me - isn't it returning the number of entries in the table?

Also why not use `COUNT(*)` in it?